src/worker.py

```diff
@@ -349,5 +352,12 @@ def _set_pcd(self, packet: PointcloudPacket) -> None:
             packet (PointcloudPacket): outputted PCD data inputted by caller
         """
         arr, byte_count = packet.points, packet.points.nbytes
+        self.logger.debug(f"Setting current pcd. num_bytes: {byte_count}")
+        if byte_count > MAX_GRPC_MESSAGE_BYTE_COUNT:
```

Comment:
For what resolutions does this get hit?

Reply:
This gets hit every single time 😭. Since the SDK does PCD using resolution aligned to the color camera, not resized width/height, each point cloud is 2073600 (1080*1920) bytes before subsampling.